File: app/src/main/java/com/example/josh/androidkotlin/MainActivity.java
package com.example.josh.androidkotlin;

import android.os.Bundle;
import android.support.v4.app.BundleCompat;
import android.support.v7.app.AppCompatActivity;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView helloText;
    @Override
    protected void onCreate(Bundle savedInstance) {
        super.onCreate(savedInstance);
        setContentView(R.layout.activity_main);

        helloText = (TextView) findViewById(R.id.helloText);
        helloText.setText("Hello Josh");
    }





}

File: kotlin/src/main/java/com/example/josh/MainActivity.kt
package com.example.josh

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.TextView

class MainActivity : AppCompatActivity() {
    var helloText: TextView? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        helloText = findViewById(R.id.helloText) as TextView?
        helloText!!.text = "Hello Josh"
    }
}
